core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -542,6 +560,8 @@ public GraphHopper init(GraphHopperConfig ghConfig) {
 
         // elevation
         this.smoothElevation = ghConfig.getBool("graph.elevation.smoothing", false);
+        this.longEdgeSamplingDistance = ghConfig.getDouble("graph.elevation.long_edge_sampling_distance", 0);
+        this.elevationSimplifyFactor = ghConfig.getDouble("graph.elevation.simplify_factor", 0);
```

Comment:
Also, should these be enabled by default? If so, what values should we use?

Reply:
Yes, for now I would keep the current settings as default (i.e. disabling 'long edge sampling'). But if we learn that it is worth the extra costs we can change it to like 60m (and 1m for max ele simplify)?

> graph.elevation.long_edge_sampling_distance

Shouldn't the default for this be Double.MAX_VALUE?

Reply:
OK sounds good, and I changed both of them to default to Double.MAX_VALUE.

Do you think that I should add sections to config-examply.yml explaining how to use these 2 new parameters (and bilinear interpolation)?

Reply:
Ah, yes. Adding this to config-example.yml is a good idea